edison/src/test/java/edu/illinois/cs/cogcomp/edison/features/factory/TestWordFeatureFactory.java

```diff
@@ -191,17 +191,17 @@ public final void testFeatureCollection() throws Exception {
             }
         }
     }
-
-    public final void testGazetteerFeatures() throws Exception {
-        System.out.println("\tTesting gazetteer features");
-        WordFeatureExtractor fex =
-                WordFeatureExtractorFactory.getGazetteerFeatureExtractor("gazetteer",
-                        new GazetteerViewGenerator("gazetteers", ViewNames.GAZETTEER));
-
-        for (TextAnnotation ta : tas) {
-            runTest(ta, fex);
-        }
-    }
+// Not needed anymore, as test of SimpleGazetteerAnnotator in SimpleGazetteerAnnotatorTest.java suffice.
```

Comment:
I dont know why Vivek did not separate the test class, but it is reasonable. After all, gazetteers are used for features right? The tests are checking the same feature, just the organization of the test code is different.